build/default.ps1

```diff
@@ -26,6 +26,13 @@ properties {
 Task Default -depends Build
 
 Task Clean {
+    $distPath = "..\dist\"
```

Comment:
since this is the clean task, if the dist. folder does not exist then surly no cleaning needs to take place? perhaps just short circuit here instead?

Reply:
It would be the case if there were just deletions in ..\dist, but there are also deletion of bin+obj. And also I think some other logic might need that dist folder and maybe is unable to create it itself, so it would need to be tested without running Clean. But there is still that bin+obj, which I think is necessary to clean. Because even on the clean slate there is a possibility, that you built from VS.

Reply:
@jakubsuchybio did u see this comment?

Reply:
yes, sorry, didn't find time to try it , will try now

Reply:
did it work? the Github Actions do not use the psake scrips.

Reply:
Yes, it works with this change too. I think it can be merged.